pkg/http/api.go

```diff
@@ -685,7 +685,7 @@ func DeleteEnvironment(host string, verifyTLS bool, apiKey string, project strin
 	params = append(params, queryParam{Key: "project", Value: project})
 	params = append(params, queryParam{Key: "environment", Value: environment})
 
-	url, err := generateURL(host, "/v3/environments/environment", nil)
+	url, err := generateURL(host, "/v3/environments/environment", params)
```

Comment:
I manually re-reviewed my PR that broke these and couldn't find any other instances

Reply:
Cool, same here -- I found `configs` by testing and during re-review I found `environments`